index.html

```diff
@@ -323,7 +323,7 @@ <h3>RevocationList2021Status</h3>
 {
   "@context": [
     "https://www.w3.org/2018/credentials/v1",
-    "https://w3id.org/vc-status-list-2021/v1"
+    "https://w3id.org/vc/status-list/v1"
```

Comment:
I think we're going to have to go back to date-stamped values here... see https://github.com/w3c-ccg/vc-status-list-2021/issues/3#issuecomment-1062428949

Reply:
Alright, done in https://github.com/w3c-ccg/vc-status-list-2021/pull/24 -- let me know what you think, @clehner.